dist/scripts/components/dropdown.js

```diff
@@ -0,0 +1,93 @@
+(function () {
+	'use strict';
+
+	// Create custom element
+	var proto = Object.create(HTMLDivElement.prototype);
+	proto.name = 'Dropdown Icon';
+
+	proto.createdCallback = function() {
+		// import the template and retrieve a document fragment from it
+		var content = document.querySelector('#dropdownIconImport').import;
+		var template = content.querySelector('#dropdownIconTemplate');
+		var clone = document.importNode(template.content, true);
+
+		// import the list items
+		var list = clone.querySelector('.f-dropdown');
```

Comment:
`.f-dropdown` was the original Foundation class. Can you change it here and in the SCSS to something more appropriate? Maybe `dropdown-menu`